main.bundle.js

```diff
@@ -2586,6 +2603,23 @@
 	  var allFoods = document.getElementById("all-foods-table");
 	  var alertBox = document.getElementById("alert-message");
 	  var holdFoods = document.getElementById("hold-foods-table");
+	  var inputFood = document.getElementById("myInput");
+
+	  inputFood.addEventListener('keyup', function () {
```

Comment:
I've been trying to get that keyup thing working all weekend -- I must be using it on the wrong element. Awesome.